src/core/remixes/convenience/UIView+RMXRemixerAPI.h

```diff
@@ -0,0 +1,60 @@
+/*
+ Copyright 2016-present Google Inc. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/** Category for UIView that adds a convenience API for Remixer. */
+@interface UIView (RMXRemixerAPI)
```

Comment:
As a general rule I'd encourage us to avoid adding convenience APIs like these into the core of Remixer unless they're critical to Remixer's use. The general concerns with extensions to Apple APIs are:

1. All extension methods must be prefixed with a framework prefix, otherwise we risk undefined behavior at runtime if the method collides with another.
2. Providing an API that accepts a view and exposes APIs can provide a clearer separation of concerns. E.g. `Remix(view).alphaVariableForKey(...)`.